app/table/columns/action-col.tsx

```diff
@@ -62,7 +67,7 @@ export const getActionsCol = <TData extends Record<string, unknown>>(
       // TODO: control flow here has always confused me, would like to straighten it out
       const actions = makeActions(row.original)
       const id = typeof row.original.id === 'string' ? row.original.id : null
-      return <RowActions id={id} actions={actions} />
+      return <RowActions id={id} actions={actions} copyIdLabel={copyIdLabel} />
```

Comment:
The changes in this file let us hook into this copy ID column logic while overriding the name to say `Copy group ID` because otherwise it's not clear what the ID is of.

<img width="324" alt="image" src="https://github.com/user-attachments/assets/7062e32d-bf98-4c73-93fb-c9a9235413ab" />